### DISCUSSION TITLE
Create partition by year  and month?

### DISCUSSION THREAD

**Discussion by scramatte** [orptech-com/laravel-migration-partition#28] [Q&A] [answered]
Hi,

I need to create partition per year and subpartitions per month   or  partition per month and subpartitions per day
Does it posible to achieve it with postgres and with your library?

Can you give me an hand?

Regards

**Comment by ArdaKaraderi** [comment, marked as answer]
Hi, 

The package does not support sub-partitioning. However, if you wish, you can add the functionality to the package yourself and we can review it.

Kind regards,
Arda

**Reply by scramatte** [reply]
Hi,

Can we sponsor implementation of such feature? 

Regards

**Reply by ArdaKaraderi** [reply]
Hi, currently we do not take sponsors. We would  be happy to review your code if your team sends a PR though.

Kind regards,
